lib/engine.js

```diff
@@ -425,8 +425,12 @@ module.exports = function (s, conf) {
               }, conf.wait_for_settlement)
             }
             else {
-              pushMessage('Buying ' + formatAsset(size, s.asset) + ' on ' + s.exchange.name.toUpperCase(), 'placing buy order at ' + formatCurrency(price, s.currency) + ', ' + formatCurrency(quote.bid - Number(price), s.currency) + ' under best bid\n')
-              doOrder()
+              if(!so.only_completed_trades){
```

Comment:
this should probably be 
```suggestion
              if(!so.notifiers.only_completed_trades){
```